### PR TITLE
PERF-640 Add test for null customer_interest_id and location_id

### PR DESCRIPTION
**Pull request by olga-andreeva-gyg**

### CODE REVIEW

File: tests/ad_templates/api/test_template.py [hunk 1]
```diff
@@ -125,3 +126,24 @@ def test_get_one_eta_ad_template(self):
...
+        self.assertEqual(response.json, {'message': 'Conversion to spreadsheet done.'})
+
+        csv_body = self._get_uploaded_csv_body(bucket)
+        csv_lines = csv_body.strip().splitlines()
+        self.assertEqual(len(csv_lines), 6)
+        self.assertEqual(AdPart.query.count(), 5)
+
+        self.assertIn(','.join(self.CSV_COLUMNS), csv_lines)
+        for ad_part in AdPart.query.all():
+            self.assertIn(self._csv_line(eta_ad_template, ad_part, eta_customer_interest, eta_location), csv_lines)
```
Comment (andrebco) [inline tests/ad_templates/api/test_template.py]: Tks for the tests. Also, if you want to make sure we have PEP8 in here you can mark `Ensure line feed at file end on Save` on Pycharm. ;-)

File: tests/ad_templates/api/test_template.py [hunk 2]
```diff
@@ -125,3 +126,24 @@ def test_get_one_eta_ad_template(self):
...
+    def test_get_null_customer_interest_id_and_location_id(self):
+        conn = self._mock_conn()
+        bucket = self._create_bucket(conn)
+
+        eta_ad_template = EtaAdTemplateFactory()
+        eta_location = EtaAdLocationFactory(eta_ad_template=eta_ad_template, location_id=0)
+        eta_customer_interest = EtaAdCustomerInterestFactory(eta_ad_template=eta_ad_template, customer_interest_id=0)
+
+        response = self.client.get('/api/ad/template')
+        self.assertEqual(response.json, {'message': 'Conversion to spreadsheet done.'})
```
Comment (olel-gyg) [inline tests/ad_templates/api/test_template.py]: shouldn't this be `Conversion to adkitchen templates done` as we're not writing to the spreadsheet anymore?

File: tms/ad_templates/resources/template.py [hunk 3]
```diff
@@ -56,9 +56,9 @@ def _get_all_eta_templates_as_df():
             AdPart.template_type,
             AdPart.landing_page_type,
             AdPart.language_id.label('gyg_language_id'),
-            EtaAdCustomerInterest.id.label('customer_interest_id'),
```
Comment (olel-gyg) [inline tms/ad_templates/resources/template.py]: 👍
